### PR TITLE
GUAC-1380 Added missing dutch translations

### PR DESCRIPTION
**Pull request by nrbrt**

### CODE REVIEW

File: guacamole/src/main/webapp/translations/nl.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     
-    "NAME" : "Nederlands",
+    "NAME" : "Nederlands (NL)",
```
Comment (mike-jumper) [inline guacamole/src/main/webapp/translations/nl.json]: Please remove the country code. Why did you add this?

File: guacamole/src/main/webapp/translations/nl.json [hunk 2]
```diff
@@ -43,7 +45,7 @@
     "CLIENT" : {
 
         "ACTION_ACKNOWLEDGE"               : "@:APP.ACTION_ACKNOWLEDGE",
-        "ACTION_CLEAR_COMPLETED_TRANSFERS" : "Wis lijst",
+        "ACTION_CLEAR_COMPLETED_TRANSFERS" : "Wis lijst Voltooide Overdrachten",
```
Comment (mike-jumper) [inline guacamole/src/main/webapp/translations/nl.json]: Is "Wis lijst" incorrect? Is there a shorter way to say this than "Wis lijst Voltooide Overdrachten"? The new user interface will not accommodate such a large button very well. The English translation, for example, is simply "Clear".

File: guacamole/src/main/webapp/translations/nl.json [hunk 3]
```diff
@@ -126,8 +129,15 @@
 
     },
 
+    "DATA_SOURCE_DEFAULT" : {
+        "NAME" : "Default (XML)"
```
Comment (mike-jumper) [inline guacamole/src/main/webapp/translations/nl.json]: Dutch for "Default" is "Default"? Elsewhere, I have seen "Standaard".

**Comment by mike-jumper** [comment]
Thanks, but:

> screenshot 2015-11-02 at 12 10 51 pm

Why such a huge number of commits?

Repeated "Delete nl.json", "Create nl.json", etc. does not really add anything to the source history. Commits should be tagged with the corresponding JIRA issue and should describe, at a high level, what is being done to the source - not just that a file is being created or deleted. That level of information is already provided by git.

Please see our [contribution guidelines](https://github.com/glyptodon/guacamole-client/blob/master/CONTRIBUTING). In particular:

> ... Commits should be made in logical units and must reference the JIRA issue number:
> 
>     $ git commit -m "GUAC-123: Message describing the specific changes made."
> 

If you use `git rebase`, you can rebase your source tree on top of our latest master such that what really *should* be only one commit actually *is* one commit, with a proper descriptive message.

File: guacamole/src/main/webapp/translations/nl.json (hunk 1 above)
Comment (nrbrt) [inline guacamole/src/main/webapp/translations/nl.json]: I translated from the English json file, which had the country code added at that time. I assumed that that was correct. It has been removed from the English json file since then.

File: guacamole/src/main/webapp/translations/nl.json (hunk 2 above)
Comment (nrbrt) [inline guacamole/src/main/webapp/translations/nl.json]: "Wis lijst" will do in that case. I tried to remove possible ambiguity.

File: guacamole/src/main/webapp/translations/nl.json (hunk 3 above)
Comment (nrbrt) [inline guacamole/src/main/webapp/translations/nl.json]: "Standaard" is also correct. Computer terminology in Dutch has a lot of English in it and "default" is widely understood as well. Some Dutch computer terminology just makes my toes curl and  "Standaard" is one of them. Dutch does not have a one-word translation for default and the Dutch word for standard is often used instead. I will  change my translation to "Standaard", no problem.

**Comment by mike-jumper** [comment]
Thanks, @nrbrt. We'll merge these via #287 as the history in this PR is a bit too crazy. The changes themselves are otherwise fine IMHO.

Please, in the future, avoid creating/deleting/creating/deleting/etc. files in a series of many commits. Changes should be really incremental. If you're having trouble getting this working on your end, I can try to help.
